### PR TITLE
Use semantic versioning

### PR DESCRIPTION
**Pull request by hvelarde**
The package is compatible with Plone 4.3 again; I fixed a couple of issues and added translations.

I think it's better to use semantic versioning for now on; if you agree, please merge this pull request and make a new release as I have a customer waiting for this feature.

IMO, as the package lacks tests, naming this new version 1.0b1 is fine.

### CODE REVIEW

**Review by frisi — APPROVED** [review]
LGTM. I'm fine with the version change.
please wait for @frapell's feedback too.

i'm not involved in the project we used this package for and i currently do not have resources to contribute.

**Comment by frapell** [comment]
Sorry guys, I'm incredibly busy these days and cannot review all of this.

@hvelarde thanks for bringing plone 4.3 back.

As per this particular PR I wouldn't like to have beta releases forever... To me, beta is software missing functionality. Flagging this as beta gives the impression of that, and it is not the case.

This product is used in several deployments and works fine.

It should certainly have tests testing everything, but I don't think it has anything to do with being beta or not...

In any case, this is my personal view, and if the majority wants to move forward, that's fine...

**Comment by hvelarde** [comment]
can you guys make a new release? you can name it whatever you like.

**Comment by frapell** [comment]
@hvelarde I have made a new release following the previous version numbering (0.13.0)

btw, I cannot push the changes zest.releaser made to master because of master being protected and apparently my user is not allowed to push.

Do you see a "Settings" for this repo where you can add me as admin so I can push these changes?

**Comment by frisi** [comment]
> Do you see a "Settings" for this repo where you can add me as admin so I can push these changes?

no, neither. seems master branches are protected on organization level (https://help.github.com/articles/about-protected-branches/)

screenshot_2018-11-08 use semantic versioning by hvelarde pull request 36 collective collective pwexpiry

 i guess this is something we need to ask the collective admins. @jensens, @mauritsvanrees do you know why collaborators can not push to master anymore?

**Comment by frapell** [comment]
The major issue is that zest.releaser commits to master directly... and now I see this "Merging blocked", so even if this is done in a branch, it cannot be merged to master anyway...

Please do not merge anything else to master until I can push the release changes

**Comment by hvelarde** [comment]
@frapell the master branch is indeed protected, but you should be able to push to it now.

**Comment by frapell** [comment]
@hvelarde I cannot:

```
remote: error: 3 of 3 required status checks are expected. At least 1 approving review is required by reviewers with write access.
To github.com:collective/collective.pwexpiry.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:collective/collective.pwexpiry.git'
```

**Comment by mauritsvanrees** [comment]
I have disabled 'Require status checks to pass before merging' in the Settings. Can you check if that helps?
We could further relax this if needed.
I had the same while making a `z3c.jbot` release [yesterday](https://github.com/zopefoundation/z3c.jbot/pull/7).

**Comment by frapell** [comment]
@mauritsvanrees Nope... this is still not letting me push to master... I do see the green "Merge pull request" button, which was not available before, but I cannot push directly to master:

```
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: At least 1 approving review is required by reviewers with write access.
To github.com:collective/collective.pwexpiry.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:collective/collective.pwexpiry.git'
```

**Comment by hvelarde** [comment]
@frapell I disabled the protection until you push your changes; let me know when you're done so we can enable it again.

this seems like a bug in GitHub; let's contact their support.

**Comment by frapell** [comment]
@hvelarde Pushed now. Thanks!

**Comment by frapell** [comment]
FWIW, I do not see a "Settings" in this repo, so I might be lacking permissions...

**Comment by hvelarde** [comment]
you are listed there, that's why I think it's a bug on GitHub:

screenshot_2018-11-08 collective collective pwexpiry

**Comment by mauritsvanrees** [comment]
In the branch protect settings, 'Require pull request reviews before merging' was still enabled. I guess that means they fall under the 'required status checks' that need to pass.

**Comment by hvelarde** [comment]
I don't think so; I use the same configuration on all of our repositories and I can still push to master. the only thing I can't do is forcing a push (`git push -f`).

**Pull request closed without merging**
